jbpm-console-ng-executor-service/jbpm-console-ng-executor-service-client/src/main/java/org/jbpm/console/ng/es/client/editors/requestlist/RequestListViewImpl.java

```diff
@@ -432,9 +433,13 @@ public void initDefaultFilters( GridGlobalPreferences preferences,
 
         filterPagedTable.addAddTableButton( createTabButton );
         presenter.setAddingDefaultFilters( false );
-        getMultiGridPreferencesStore().setSelectedGrid( REQUEST_LIST_PREFIX + "_0" );
-        filterPagedTable.setSelectedTab();
-        applyFilterOnPresenter( REQUEST_LIST_PREFIX + "_0" );
+        Scheduler.get().scheduleDeferred(new Scheduler.ScheduledCommand() {
+            @Override
+            public void execute() {
+                getMultiGridPreferencesStore().setSelectedGrid( REQUEST_LIST_PREFIX + "_0" );
+                filterPagedTable.setSelectedTab();
```

Comment:
The applyFilterOnPresenter was called twice. It's enough the setSelectedTab which has the filter call on the command